tools/cmake/Findlibwebsockets.cmake

```diff
@@ -6,28 +6,10 @@
 # LIBWEBSOCKETS_FOUND, If false, do not try to use libWebSockets
 #
 # This currently works probably only for Linux
+set(FPHSA_NAME_MISMATCHED 1) # Suppress warnings, see https://cmake.org/cmake/help/v3.17/module/FindPackageHandleStandardArgs.html
+include(FindPkgConfig)
 
-FIND_PATH ( LIBWEBSOCKETS_INCLUDE_DIR libwebsockets.h
-    /usr/local/include
-    /usr/include
-)
+pkg_search_module(LIBWEBSOCKETS REQUIRED libwebsockets)
```

Comment:
This requires pkg-config and is only available for Linux.
The previous cmake file was multi-platform.
And it should stay that way.